src/config/entities/configuration.ts

```diff
@@ -209,4 +209,11 @@ export default () => ({
   safeWebApp: {
     baseUri: process.env.SAFE_WEB_APP_BASE_URI || 'https://app.safe.global',
   },
+  swaps: {
+    api: {
+      1: 'https://api.cow.fi/mainnet',
+      100: 'https://api.cow.fi/xdai',
+      58008: 'https://api.cow.fi/sepolia',
```

Comment:
The chain ID of Sepolia is `11155111`.

Reply:
It is 😅 I don't know how I came up with that number. Changed in [ddaf881](https://github.com/safe-global/safe-client-gateway/pull/1233/commits/ddaf881310b8a594df12d82f1ddbebe449cb33d5)